src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/__tests__/ArtistWrapper.test.tsx

```diff
@@ -34,22 +38,12 @@ describe("ArtistWrapper", () => {
     expect(component.state.isExpanded).toBe(!isExpanded)
   })
 
-  /**
```

Comment:
cc @ashleyjelks I reworked this spec a little bit to avoid the randomness (but kept one to be expressive about the different sections).

It seemed like this test basically tests that calling lodash `random` twice gives you different results, which I don't think is really needed as a spec. Plus, it's important to keep specs as deterministic as possible to avoid CI flakiness and spurious failures.